### PR TITLE
[MIG] Migrates l10n_nl_partner_name from 8.0 to 10.0

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

File: l10n_nl_partner_name/README.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    Use Dutch conventions for partner names:
+    - have infixes
+    - have initials
+    - split first and last name (provided by partner_firstname)
+
+Contributors
+------------
+
+* Holger Brunn <hbrunn@therp.nl>
+* Ronald Portier <ronald@therp.nl>
```
Comment (NL66278) [inline l10n_nl_partner_name/README.rst:16]: And you can add your own name here

File: l10n_nl_partner_name/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (NL66278) [inline l10n_nl_partner_name/__init__.py]: Use shortened headers

File: l10n_nl_partner_name/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    'version': '10.0.1.0.0',
+    'author': 'Therp BV, Odoo Community Association (OCA)',
+    'category': 'Contact management',
+    'depends': [
+        'partner_firstname',
+    ],
+    'data': [
+        'view/res_partner.xml',
+    ],
+    'js': [],
```
Comment (NL66278) [inline l10n_nl_partner_name/__manifest__.py]: Delete empty keys

File: l10n_nl_partner_name/model/__init__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from . import res_partner
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (NL66278) [inline l10n_nl_partner_name/model/__init__.py]: Delete vim comment lines

File: l10n_nl_partner_name/model/res_partner.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+##############################################################################
+from mako.template import Template
+from odoo import models, fields, api
+
+
+class ResPartner(models.Model):
+    """Extend res.partner with extra fields for Dutch names."""
+    _inherit = 'res.partner'
+
+    initials = fields.Char(size=16)
```
Comment (NL66278) [inline l10n_nl_partner_name/model/res_partner.py]: No need for size anymore

**Comment by daramousk** [comment]
@NL66278 Implemented everything

**Comment by daramousk** [comment]
Closing this as it is contained in #110

**Pull request closed without merging**